mono/mini/interp/interp.c

```diff
@@ -4494,6 +4494,7 @@ interp_exec_method_full (InterpFrame *frame, ThreadContext *context, guint16 *st
 		MINT_IN_CASE(MINT_ICALL_PPI_V)
 		MINT_IN_CASE(MINT_ICALL_PII_P)
 		MINT_IN_CASE(MINT_ICALL_PPII_V)
+			frame->ip = ip;
```

Comment:
hmm, should we have this at the loop header, so `frame->ip` is always accurate?

Reply:
@lewurm doesn't look like this is possible with the optimized switch-case